index.html

```diff
@@ -4,63 +4,219 @@
+                    </div>
+                    <p class="content">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <nav>
+                        <ul>
+                            <li>
+                                <a href="" target="_blank">
+                                    <img src="./assets/img-block1/html.png" alt="">
+                                </a>
+                            </li>
+                            <li>
+                                <a href="" target="_blank">
+                                    <img src="./assets/img-block1/css.png" alt="">
+                                </a>
+                            </li>
+                            <li>
+                                <a href="" target="_blank">
+                                    <img src="./assets/img-block1/javascript.png" alt="">
+                                </a>
+                            </li>
+                        </ul>
+                    </nav>
+                    <div>
+                        <button type="button" class="see_button">See Project</button>
+                    </div>
+                </div>                
+            </section>
+    <!--card_4 starts from here-->
+            <section class="picture_container">
+                    <img src="./assets/pictures/picture4.jpg" alt=""/>
+                <div class="info_container">   
+                    <h1 class="title">Multi-Post<br>Stories</h1>
+                    <div class="tonic_1">
+                        <img src="./assets/img-block1/Frame1.png" alt="">   
+                    </div>
+                    <p class="content">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <nav>
+                        <ul>
+                            <li>
+                                <a href="" target="_blank">
+                                    <img src="./assets/img-block1/html.png" alt="">
+                                </a>
+                            </li>
+                            <li>
+                                <a href="" target="_blank">
+                                    <img src="./assets/img-block1/css.png" alt="">
+                                </a>
+                            </li>
+                            <li>
+                                <a href="" target="_blank">
+                                    <img src="./assets/img-block1/javascript.png" alt="">
+                                </a>
+                            </li>
+                        </ul>
+                    </nav>
+                    <div>
+                        <button type="button" class="see_button">See Project</button>
+                    </div>
+                </div>                
+            </section>
+<!--about myself section from here-->
+            <div id="myself_div">    
```

Comment:
- [ ] You should use `<section>` element instead of a `<div>` here for semantic reasons.

Reply:
In your previous [OPTIONAL] comment, you recommend choosing just one main or section tag.

Reply:
The `about me` section is just a section that can stand on its own like the `works` sections. Therefore, it is semantically more appropriate to use the `<section>` tag to instead of a generic `<div>` tag. Kindly check [this](https://www.w3schools.com/html/html_layout.asp) reference for an HTML page layout using semantic tags.

Reply:
I appreciate your suggestion. Now I got it! 👍